packages/devtools_app/lib/src/debugger/flutter/debugger_controller.dart

```diff
@@ -80,6 +80,16 @@ class DebuggerController extends DisposableController
 
   ValueListenable<String> get exceptionPauseMode => _exceptionPauseMode;
 
+  final ValueNotifier<bool> _librariesVisible = ValueNotifier(false);
```

Comment:
remove type on LHS

Reply:
Do we follow that pattern for fields as well? (I know we do for locals)

Reply:
yeah for final initialized fields we have been omitting LHS types

Reply:
ok, will remove here